websites/G/GenoAnime/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "DayAndNight",
+    "id": "424072110647017482"
+  },
+  "url": "genoanime.com",
+  "description": {
+    "en": "Enjoy the our website, view any anime episode and movie"
+  },
+  "service": "GenoAnime",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/bkcZfjM.jpeg",
```

Comment:
logo must be 512x512

Reply:
![image](https://user-images.githubusercontent.com/79012803/107865729-18533280-6e40-11eb-8f8c-b991d65faf17.png)

![image](https://user-images.githubusercontent.com/79012803/107865744-40db2c80-6e40-11eb-9b0b-bd5e65acdcc5.png)